{{cookiecutter.github_project_name}}/src/plugin.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  Application, IPlugin
+} from '@phosphor/application';
+
+import {
+  Widget
+} from '@phosphor/widgets';
+
+import * as yourCode from './example';
+
+import {
+  JUPYTER_EXTENSION_VERSION
+} from './version';
+
+import {
+  IJupyterWidgetRegistry
+ } from '@jupyter-widgets/base';
+
+
+const EXTENSION_ID = '{{ cookiecutter.jlab_extension_id }}';
+
+
+/**
+ * The example plugin.
+ */
+const examplePlugin: IPlugin<Application<Widget>, void> = {
```

Comment:
Notes about this:

- We only use `@phosophor/*` for its interfaces, so it is a `devDependency` that doesn't end up in the notebook extension.
- Because we use the generic application, this is already API stable and could work in any Phosphor application